.github/workflows/deploy_tre.yml

```diff
@@ -44,53 +38,160 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
 
-      - name: Install Terraform
-        uses: hashicorp/setup-terraform@v1
+      - name: Set up Docker BuildKit
+        uses: docker/setup-buildx-action@v1
+
+      - name: Login to Container Registry
+        uses: docker/login-action@v1
         with:
-          terraform_version: 1.0.5
+          registry: ${{ secrets.ACTIONS_ACR_NAME }}.azurecr.io
+          username: ${{ secrets.ACTIONS_ACR_NAME }}
+          password: ${{ secrets.ACTIONS_ACR_PASSWORD }}
 
-      - name: Deploy TRE
+      - name: Set Variables
+        run: |
+          echo "USER_UID=$(id -u)" >> $GITHUB_ENV
+          echo "USER_GID=$(id -g)" >> $GITHUB_ENV
```

Comment:
Is this only used once for building the devcontainer? 
If yes, do you really need this here and insert it to the env of all steps?

Reply:
I've pulled these 2 inline with the docker build. These were left over when I was refactoring.